cmd/syncthing/monitor.go

```diff
@@ -181,6 +186,43 @@ func copyStderr(stderr io.ReadCloser, dst io.Writer) {
 					panicFd.WriteString(line)
 				}
 				stdoutMut.Unlock()
+
+				if optPanicReporting {
```

Comment:
I guess it doesn't make charm to print it either way so that we get pinged about a bug. After all if somebody creates an issue it means that it is a sever problem that affects his usage of Syncthing.